idiomatic_ruby/array_of_strings_symbols.md

```diff
@@ -3,13 +3,13 @@
 **Array of Strings**
 
 ```ruby
-%w[Michael Jonh Mary Stuart]
+%w(Michael Jonh Mary Stuart)
```

Comment:
Probably we could like this [link](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces). What do you think, @rthbound ?